tests/app/test_jinja_filters.py

```diff
@@ -90,6 +92,20 @@ def test_format_date_time_in_gmt(self):
             format_value, "<span class='date'>28 October 2018 at 11:59</span>"
         )
 
+    def test_format_date_using_london_timezone(self):
+        # Given a London timezone
+        london_timezone = timezone("Europe/London")
+        loc_dt = london_timezone.localize(datetime(2021, 8, 3, 00, 15, 0))
+
+        # When
+        with self.app_request_context("/"):
+            format_value = format_datetime(self.autoescape_context, loc_dt)
+
+        # Then
+        self.assertEqual(
+            format_value, "<span class='date'>3 August 2021 at 00:15</span>"
+        )
```

Comment:
This test is incorrect as it is passing a London date time to format datetime instead of UTC datetime. Since the existing two tests have the date in the output, we should:
- update those to test date **and** time
- The date we pass should be UTC since that is the actual behaviour. Calling `datetime()` would not have timezone info by default.
- For testing GMT, I think it makes sense to do two dates, as I found it confusing without.

We should update the existing tests to something like this, which test both the date and time in terms of BST/GMT.
```py
    def test_format_date_time_in_bst(self):
        # Given a date after BST started
        date_time = datetime(2018, 3, 29, 23, 59, 0).replace(tzinfo=timezone.utc)

        # When
        with self.app_request_context("/"):
            format_value = format_datetime(self.autoescape_context, date_time)

        # Then
        self.assertEqual(
            format_value, "<span class='date'>30 March 2018 at 00:59</span>"
        )

    def test_format_date_time_in_gmt(self):
        # Given
        test_data = {
            datetime(2018, 10, 28, 00, 15, 0): "28 October 2018 at 01:15",
            # Clocks go back on 29th Oct 2018
            datetime(2018, 10, 29, 00, 15, 0): "29 October 2018 at 00:15"
        }
        for date_time, expected_value in test_data.items():
            with self.subTest(
                date_time=date_time, expected_valye=expected_value,
            ):
                date_time = date_time.replace(tzinfo=timezone.utc)
                # When
                with self.app_request_context("/"):
                    format_value = format_datetime(self.autoescape_context, date_time)

                # Then
                self.assertEqual(
                    format_value, f"<span class='date'>{expected_value}</span>"
                )
```